src/test_l1_tvSettings.c

```diff
@@ -11196,7 +11197,7 @@ void test_l1_tvSettings_negative_GetColorTemp_R_post_offset_onSource (void)
         platformFlag = false;
         for (unsigned int j = 0; j < colorTempCount ; j++ )
         {
-            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", i);
+            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", j);
             colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
             if(colorTempValue == (tvColorTemp_t)i)
             {
```

Comment:
"In line number 11210, are we supposed to pass the index i instead of colorTempValue? This is because we intend to pass the unsupported value."

Reply:
updated